spec/imsc-hrm.html

```diff
@@ -556,12 +559,16 @@ <h2>Paint Text</h2>
       <p class='note'>In the case where a property is <em>prohibited</em> in a profile of [[IMSC]], the computed value of the property
       specified in [[ttml2]] can be used.</p>
 
-      <p class='note'>While one-to-one mapping between characters and typographical glyphs is generally the rule in some scripts,
-      e.g. latin script, it is the exception in others. For instance, in arabic script, a character can yield multiple glyphs
-      depending on its position in a word. The Hypothetical Render Model always assumes a one-to-one mapping, but reduces the
-      performance of the glyph buffer for scripts where one-to-one mapping is not the general rule (see GCpy below).</p>
+      <p class='note'>While one-to-one mapping between code points and glyphs (as defined in [[i18n-glossary]])
```

Comment:
see above note. You can just `<a>` or `[= =]` markup the terms and respec will link them to our glossary.

Reply:
Since this spec defines "glyph" itself I think that will take the reader to the wrong definition. That's another good reason for changing the tuple definition term name from "glyph" in the Paint Text section to some other non-ambiguous name.

Happy to do that for code points though.

Reply:
Done where I can, in 723cd39.